src/main/java/org/broadinstitute/barclay/argparser/CommandLineProgramProperties.java

```diff
@@ -25,7 +25,22 @@
      * @return an example command line for this program
      */
     String usageExample() default "The author of this program hasn't included any example usage, please complain to them.";
+
+    /**
+     * Assign this command line program to an (application-defined) program group for use in managing help output
+     * and documentation.
+     * @return CommandLineProgramGroup for this command line program
+     */
     Class<? extends CommandLineProgramGroup> programGroup();
 
+    /**
+     * Property for use by client applications that may want to hide some command like programs from interactive
+     * command line help output. This property is for use by consuming applications only. Barclay does not use
+     * or enforce this property for any purpose (including generated documentation, which is governed separately by
+     * the {@link org.broadinstitute.barclay.help.DocumentedFeature} annotation).
+     *
+     * @return true if this command line program should be omitted by the Barclay client application from interactive
+     * help output
```

Comment:
I would change this for `help output; {@code false} otherwise`, but again this is personal taste.

Reply:
Done.